test/openscap_parser/test_result_file_test.rb

```diff
@@ -416,6 +416,294 @@ def setup
     end
   end
 
+  context 'Verify the parser works the same with ARF and XCCDF files' do
+    {
+      'xccdf' => OpenscapParser::TestResultFile.new(file_fixture('ssg-rhel9-ds-xccdf.results.xml').read),
+      'arf' => OpenscapParser::TestResultFile.new(file_fixture('ssg-rhel9-ds-arf.xml').read)
+    }.each do |report_type, report|
```

Comment:
I like how you set this up! I guess we're technically duplicating the XCCDF tests since we already have tests for a different XCCDF file but I think it's fine since the purpose of this is testing an ARF file against an XCCDF file.

Reply:
Thanks, surely someone should go through the tests and remove the duplicate tests and create the missing tests for ARF and XCCDF in this less duplicated way.